motorlib/nozzle.py

```diff
@@ -45,6 +45,11 @@ def getExitArea(self):
     def getExitPressure(self, k, inputPressure):
         """Solves for the nozzle's exit pressure, given an input pressure and the gas's specific heat ratio."""
         return fsolve(lambda x: (1/self.calcExpansion()) - eRatioFromPRatio(k, x / inputPressure), 0)[0]
+        
+    def getSeparationPressureRatio(self):
+        """Finds the pressure ratio at which the nozzle is predicted to ingest a shockwave and flow separate using the Summerfield criteria.
+        https://ntrs.nasa.gov/api/citations/19840011402/downloads/19840011402.pdf"""
+        return 0.4
```

Comment:
From the article it seems like there might be a reason for some users to want to configure this. Think it should be a configuration value?